WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -432,6 +433,22 @@ private extension NewOrderViewModel {
             .assign(to: &$navigationTrailingItem)
     }
 
+    /// Updates the notice based on the `orderSynchronizer` sync state.
+    ///
+    func configureSyncErrors() {
+        orderSynchronizer.statePublisher
+            .map { state in
+                switch state {
+                case .error(let error):
+                    DDLogError("⛔️ Error syncing new order remotely: \(error)")
+                    return NoticeFactory.syncOrderErrorNotice(with: self.orderSynchronizer)
```

Comment:
I think this can potentially leak 😅

Reply:
Oh shoot, you're right; I missed that remaining `self` reference. I'll fix that tomorrow!